classes/MyTableAdmin.php

```diff
@@ -223,57 +225,75 @@ protected function outputField(array $field, $key, array $record, array $options
             $output .= $this->outputForeignId(
                 "fields[$key]",
                 'SELECT id,' . Tools::escapeDbIdentifier($comment['foreign-column']) . ' FROM ' . Tools::escapeDbIdentifier(TAB_PREFIX . $comment['foreign-table']),
-                $value, ['class' => 'form-control', 'id' => $input['id'], 'exclude' => (TAB_PREFIX . $comment['foreign-table'] == $this->table ? [$value] : [])]);
+                $value,
+                ['class' => 'form-control', 'id' => $input['id'], 'exclude' => (TAB_PREFIX . $comment['foreign-table'] == $this->table ? [$value] : [])]
+            );
             $input = false;
             $field['type'] = null;
         }
         switch ($field['type']) {
-            case 'tinyint': case 'smallint': case 'int': case 'mediumint': case 'bigint': case 'year':
+            case 'tinyint':
+            case 'smallint':
+            case 'int':
+            case 'mediumint':
+            case 'bigint':
+            case 'year':
+                // TODO fix Binary operation "+=" between arrays results in an error.
                 $input += ['type' => 'number', 'step' => 1, 'class' => 'form-control'];
                 if ($field['key'] == 'PRI') {
                     $input['readonly'] = 'readonly';
                     $input = '<div class="input-group">' . Tools::htmlInput("fields[$key]", false, $value, $input)
-                            . '<span class="input-group-btn"><button class="btn btn-secondary btn-id-unlock" type="button" title="' . $this->translate('Unlock') . '"><i class="glyphicon glyphicon-lock fa fa-lock" aria-hidden="true"></i></button></span></div>';
+                        . '<span class="input-group-btn"><button class="btn btn-secondary btn-id-unlock" type="button" title="' . $this->translate('Unlock') . '"><i class="glyphicon glyphicon-lock fa fa-lock" aria-hidden="true"></i></button></span></div>';
                 }
                 break;
             case 'date':
-                $input += [/*'type' => 'date',*/ 'class' => 'form-control input-date'];
+                // TODO fix Binary operation "+=" between arrays results in an error.
+                $input += [/* 'type' => 'date', */ 'class' => 'form-control input-date'];
```

Comment:
@crs2 Co má dělat to +=  na typu array?

Reply:
Po tom příkazu má být $input['class'] = 'form-control input-date'.
Dobrý postřeh - v tomhle případě je chybné pořadí - správně má být buď:
$input['class'] = 'form-control input-date';
nebo
$input = [/* 'type' => 'date', */ 'class' => 'form-control input-date'] + $input;
Jsem pro první verzi, která je čitelnější (určitě pro přiřazení jednoho prvku do pole) a druhou pro přiřazení více než jednoho prvku do pole.

Reply:
Aha. A + mezi array je union. Good. Slávek